server/lib/config/pbench-server.cfg.example

```diff
@@ -48,14 +48,14 @@ tasks = pbench-sync
 satellite-host = satellite-one.example.com
 satellite-prefix = ONE
 satellite-lock = pbench-sync-satellite-%(satellite-prefix)s.lock
-satellite-archive = /pbench/archive/fs-version-001
+satellite-archive = /pbench/archive/fs-version-002
```

Comment:
This should not be changed. The archive fs-version is unrelated to the reception area fs-version.